File: src/main/java/com/fwu/lc_core/LcCoreApplication.java
package com.fwu.lc_core;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import reactor.core.publisher.Hooks;

@SpringBootApplication
@EnableJpaRepositories
public class LcCoreApplication {
    public static void main(String[] args) {
        SpringApplication.run(LcCoreApplication.class, args);
    }
}

File: src/main/kotlin/com/fwu/lc_core/LcCoreApplication.kt
package com.fwu.lc_core

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@SpringBootApplication
class LcCoreApplication

fun main(args: Array<String>) {
    runApplication<LcCoreApplication>(*args)
}

@RestController
class HtmlController {
    @GetMapping("/hello")
    fun hello(name: String?): String {
        val iName: String = name ?: "World"
		return "Hello, $iName!"
    }
}